app/controllers/entry_requests_controller.rb

```diff
@@ -2,13 +2,23 @@ class EntryRequestsController < ApplicationController
   before_action :require_resort_or_group_leader
 
   def update
-    user = User.find params[:user_id]
+    begin
+      create_or_update_point_request
+    rescue ActiveRecord::RecordInvalid, RecordNotFound
+      return head :unprocessable_entity
+    end
+
+    head :ok
+  end
```

Comment:
Here you can leave the `begin`/`end` also.
```suggestion
    create_or_update_point_request
    head :ok  
  rescue ActiveRecord::RecordInvalid, RecordNotFound
    head :unprocessable_entity
  end
```